packages/twig/cypress/e2e/navigation/navigation.cy.js

```diff
@@ -1,3 +1,5 @@
+/* @ESLINT-DEBT During new eslint refactoring this file was omitted because of heavy type refactoring */
```

Comment:
Ideally these ESLINT-DEBT annotations should reference the issue that describes why they were placed and how they should be fixed.

Reply:
@GGKapanadze to create the issue and then add the issue number to the comment.